OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseDiscussionPostsThreadFragment.java

```diff
@@ -124,7 +125,10 @@ public void onCreate(Bundle savedInstanceState) {
     @Nullable
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
-        return inflater.inflate(R.layout.fragment_discussion_thread_posts, container, false);
+        final View view = inflater.inflate(R.layout.fragment_discussion_thread_posts, container, false);
+        // Being initialized here so that its non-null when base class's onViewCreated is called
```

Comment:
Following doc looks more readable.
// Initializing `errorNotification` here to get it non-null in base class's `onViewCreated` callback

Reply:
how about this?

```java
// Initializing errorNotification here, so that its non-null value can be used in the parent class's `onViewCreated` callback
```